eng/Version.Details.xml

```diff
@@ -214,5 +214,9 @@
       <Uri>https://github.com/dotnet/hotreload-utils</Uri>
       <Sha>25b814e010cd4796cedfbcce72a274c26928f496</Sha>
     </Dependency>
+    <Dependency Name="System.Runtime.Numerics.TestData" Version="6.0.0-beta.21314.1">
```

Comment:
This was merged in https://github.com/dotnet/runtime-assets/pull/143

However, I am having trouble restoring this package locally. I get the following error when I do `build.cmd -vs System.Runtime`:

```
D:\repos\runtime_2\src\libraries\System.Runtime\tests\System.Runtime.Tests.csproj : error NU1101: Unable to find package System.Runtime.NumericsTestData. No packages exist with this id in source(s): dotnet-eng, dotnet-public, dotnet-tools, dotnet6, dotnet6-transport, richnav [D:\repos\runtime_2\src\libraries\System.Runtime\System.Runtime.sln]
```

What am I doing wrong? @safern @ViktorHofer

Reply:
The PackageId is wrong: `System.Runtime.NumericsTestData`. It should instead be `System.Runtime.Numerics.TestData`.

Reply:
But I have specified Name here to be `System.Runtime.Numerics.TestData`. Where is `PackageId` specified?!?!

Reply:
> Where is PackageId specified?!?!

That is in the `System.Runtime.Tests.csproj`. That is where NuGet grabs the package to restore. This file is merely informational for DARC to update dependency versions when a new package is published.

Reply:
Ok never mind, I think I found it

Reply:
Right, you need the following three changes to consume the package and keep it up-to-date:
- The Version.Details.xml entry which points to the runtime-assets repo
- The Versions.props property with the same name as the PackageId just without dots, `SystemRuntimeNumericsTestDataVersion`.
- The PackageReference in the test project that uses the property above to restore and use the test data package.